projects/dotcms-field-elements/src/utils/test/utils.tsx

```diff
@@ -12,5 +12,24 @@ export const dotTestUtil = {
         filledRequired: ['dot-required', 'dot-valid', 'dot-touched', 'dot-dirty'],
         filledRequiredPristine: ['dot-required', 'dot-valid', 'dot-untouched', 'dot-pristine'],
         touchedPristine: ['dot-valid', 'dot-pristine', 'dot-touched']
+    },
+    triggerStatusChange: (
+        pristine: boolean,
```

Comment:
too many params? maybe object?

Reply:
agree, but create an object will add more "noise" to the calls in the test.